v0/destinations/pinterest_tag/data/pinterestCommonConfig.json

```diff
@@ -0,0 +1,62 @@
+[
+  {
+    "destKey": "event_name",
+    "sourceKeys": "event"
+  },
+  {
+    "destKey": "event_time",
+    "sourceKeys": "timestamp",
+    "sourceFromGenericMap": true,
+    "required": true,
+    "metadata": {
+      "type": "secondTimestamp"
+    }
+  },
+  {
+    "destKey": "event_source_url",
+    "sourceKeys": "pageUrl",
+    "sourceFromGenericMap": true
+  },
+  {
+    "destKey": "action_source",
+    "sourceKeys": [
+      "traits.action_source",
+      "context.traits.action_source",
+      "traits.actionSource",
+      "context.traits.actionSource",
+      "properties.action_source",
+      "properties.actionSource",
+      "channel"
+    ],
+    "required": true
+  },
+  {
+    "destKey": "app_name",
+    "sourceKeys": ["properties.appName", "context.app.name"]
+  },
+  {
+    "destKey": "app_version",
+    "sourceKeys": ["properties.appVersion", "context.app.version"]
+  },
+  {
+    "destKey": "device_brand",
```

Comment:
Should the priority of user properties be more than contextual properties for device related values?

Reply:
good find. @utsabc WDYT?